partials/new_device_login.html

```diff
@@ -1,4 +1,8 @@
 <% extends "partials/base/base_text_only.html" %>
 
 <% block heading %>{{t "New sign-in to Firefox"}}<% endblock %>
-<% block text_primary %>{{{ device }}}<br/>{{ timestamp }}<% endblock %>
+<% block text_primary %>{{{ device }}}<br/>
+<span style="display:block;">{{t "IP address: %(ip)s" }}</span>
+{{ location }}
```

Comment:
Is there an advantage to a `span` with `display:block` over a block level element like a `div` or `p`? Can you wrap location with a span/div as well, for consistency?

@ryanfeeley - Are location and timestamp supposed to have labels?